### PR TITLE
Add Homework 04 - Copy of Homework 03

### PR DESCRIPTION
**Pull request by Teodor1331**
Homework 04 - Teodor Dishanski, XII B, Number 19

### CODE REVIEW

File: .github/workflows/homework_04_19_teodor_dishanski.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        cd homework_04/19_teodor_dishanski
+        pytest
+    - name: Run All Linters
+      run: |
+        cd homework_04/19_teodor_dishanski
+        pylint *.py
+        pylama *.py
+        pycodestyle *.py
+        pydocstyle *.py
+        mypy *.py
```
Comment (Vic-Dim) [inline .github/workflows/homework_04_19_teodor_dishanski.yaml:49]: Too many linters. Maybe this is little bit unnecessary

File: homework_04/19_teodor_dishanski/Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+make:
```
Comment (Vic-Dim) [inline homework_04/19_teodor_dishanski/Makefile:1]: Nice makefile

File: homework_04/19_teodor_dishanski/task.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
+"""The file contains the logic of the task from Homework 3."""
```
Comment (Vic-Dim) [inline homework_04/19_teodor_dishanski/task.py:1]: Self-documenting code + coments, good

File: homework_04/19_teodor_dishanski/test_circle.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+"""Test the logic in the Circle class."""
```
Comment (Vic-Dim) [inline homework_04/19_teodor_dishanski/test_circle.py:1]: Tests cover all the possible cases

File: homework_04/19_teodor_dishanski/test_point.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
+"""Test the logic in the Point class."""
```
Comment (Vic-Dim) [inline homework_04/19_teodor_dishanski/test_point.py:1]: It's nice that you have separate tests for the Point class

**Review by Vic-Dim — APPROVED** [review]

File: .github/workflows/homework_04_19_teodor_dishanski.yaml (hunk 1 above)
Comment (marti456) [reply .github/workflows/homework_04_19_teodor_dishanski.yaml:49]: Nice work

**Comment by marti456** [comment]
OK

**Pull request merged**